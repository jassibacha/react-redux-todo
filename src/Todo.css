.Todo {
    display: flex;
    justify-content: space-between;
    align-items: center;
    align-content: center;
    flex-wrap: nowrap;
    padding: 10px;
    background-color: #f9f6ee;
    border-bottom: 1px solid #d4cebd;
}
