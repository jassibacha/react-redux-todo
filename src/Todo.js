import React from 'react';
import './Todo.css';

const Todo = ({ task, id, deleteTodo }) => {
    return (
        <div
            className="Todo d-flex justify-content-between align-items-center border p-3 mb-2"
            key={id}
        >
            <div className="Todo-task">{task}</div>
            <button className="btn btn-danger btn-sm" onClick={deleteTodo}>
                X
            </button>
        </div>
    );
};

export default Todo;
