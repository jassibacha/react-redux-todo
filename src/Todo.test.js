import { render, screen } from '@testing-library/react';
import Todo from './Todo';

it('renders without crashing', function () {
    render(<Todo task="A Task" id="test-id" deleteTodo={() => {}} />);
});

it('matches snapshot', function () {
    const { asFragment } = render(
        <Todo task="A Task" id="test-id" deleteTodo={() => {}} />
    );
    expect(asFragment()).toMatchSnapshot();
});
