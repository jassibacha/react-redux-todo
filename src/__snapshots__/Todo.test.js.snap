// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="Todo d-flex justify-content-between align-items-center border p-3 mb-2"
  >
    <div
      class="Todo-task"
    >
      A Task
    </div>
    <button
      class="btn btn-danger btn-sm"
    >
      X
    </button>
  </div>
</DocumentFragment>
`;
